pages/index.js

```diff
@@ -18,6 +42,7 @@ export default function HomePage() {
           <TransactionItem transaction={transaction} key={transaction._id} />
         ))}
       </TransactionsList>
+      <Form onSubmit={handleSubmit}/>
```

Comment:
hmm, you don't even pass the submitting prop to your Form, therefore the buttons are never disabled...